pkgs/development/python-modules/python-fuse/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib
+, pkgs
+, pythonPackages
```

Comment:
I think it's better to have explicit dependencies, so please replace "pkgs" and "pythonPackages" with specific packages. (Unless there is a new style going on in nixpkgs python packaging that I've missed.)